tests/contrib/validation/cerberus/test_validator_cerberus.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+from unittest import TestCase
+import six
+from slugify import slugify
+
+from spidermon.contrib.validation import CerberusValidator
+
+
+class SchemaTestCaseMetaclass(type):
```

Comment:
I don't like this meta-class way to define the tests. I prefer the tests to have more explicit intentions.
Also, as we are creating an entire new feature, we could use pytest from the beginning.


For example:
```
def test_schema_in_invalid_format_provided():
    with pytest.raises(DocumentError):
        validator = CerberusValidator("I_AM_NOT_A_MAPPING")
```

Create the other tests like functions. It would be easier to understand what is being tested.

Reply:
Renne, I am good with that. Creating functions for each test. But, if we could be able to refactor the loop approach that is working right now with unit tests in  PyTest. Then, it it would be really really efficient what say? 

I want to work towards that only. Otherwise, for this, I will just create their functions. 
Also, I think it would raise a `ValueError`, No? (In your code snippet)